mtec/integrationpluginmtec.json

```diff
@@ -0,0 +1,106 @@
+{
+    "name": "MTec",
+        "displayName": "M-Tec",
+        "id": "07cd316b-1e2c-40cf-8358-88d7407506ae",
+        "vendors": [
+        {
+            "name": "MTec",
+            "displayName": "M-Tec",
+            "id": "04d3fa7c-e469-4a79-a119-155426e5a846",
+            "thingClasses": [
+            {
+                "name": "mtec",
+                "displayName": "MTec",
+                "id": "451e38d8-50d5-4ae9-8d9f-21af9347128d",
+                "createMethods": ["user"],
+                "interfaces": ["connectable"],
+                "paramTypes":[
+                {
```

Comment:
Just for the eye, try to keep the indentations consistent in the entire document, changing or updating later is much easier if they are correct.

Reply:
You're referring to the missing space between : and [? Or did I miss something else?

Reply:
Yes, it's  bit pedantic, I know, but much easier to read :-)

Reply:
No problem, easy if it's just that!